src/platform/forms-system/test/js/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -563,6 +571,81 @@ describe('<ReviewCollapsibleChapter>', () => {
     wrapper.unmount();
   });
 
+  it('does not display page if all fields are hidden on review', () => {
+    const pages = [
+      {
+        pageKey: 'test1',
+        title: '',
+        schema: {},
+        uiSchema: {},
+      },
+    ];
+    const chapterKey = 'test';
+    const chapter = {};
+    const form = {
+      pages: {
+        test1: {
+          editMode: false,
+          schema: {
+            properties: {
+              hiddenOnSchema: {
+                type: 'boolean',
+                'ui:hidden': true,
+              },
+              collapsedOnSchema: {
+                type: 'boolean',
+                'ui:collapsed': true,
+              },
+              hiddenByUiHideOnReview: {
+                type: 'boolean',
+              },
+              hiddenByUiHideOnReviewFunction: {
+                type: 'boolean',
+              },
+              hiddenByUiHideOnReviewIfFalse: {
+                type: 'boolean',
+              },
+            },
+          },
+          uiSchema: {
+            hiddenByUiHideOnReview: {
+              'ui:options': {
+                hideOnReview: true,
+              },
+            },
+            hiddenByUiHideOnReviewFunction: {
+              'ui:options': {
+                hideOnReview: () => true,
+              },
+            },
+            hiddenByUiHideOnReviewIfFalse: {
+              'ui:options': {
+                hideOnReviewIfFalse: true,
+              },
+            },
```

Comment:
The name here suggests that this _wouldn't_ cause this property to be hidden.

Reply:
Yeah, I thought that these names would make things more clear, but it's definitely not helping. `hideOnReviewIfFalse` is already confusing enough as is.